marshmallow/utils.py

```diff
@@ -334,16 +333,34 @@ def callable_or_raise(obj):
     return obj
 
 
-def get_func_args(func):
-    """Given a callable, return a tuple of argument names. Handles
+def get_func_args2(func):
+    """Given a callable, return a list of argument names. Handles
     `functools.partial` objects and class-based callables.
+    This is the orignial Python 2 version
     """
     if isinstance(func, functools.partial):
-        return inspect.getargspec(func.func).args
+        return list(inspect.getargspec(func.func).args)
     if inspect.isfunction(func) or inspect.ismethod(func):
-        return inspect.getargspec(func).args
-    # Callable class
-    return inspect.getargspec(func.__call__).args
+        return list(inspect.getargspec(func).args)
+    if callable(func):
+        return list(inspect.getargspec(func.__call__).args)
+
+
+def get_func_args3(func):
+    """Given a callable, return a list of argument names. Handles
+    `functools.partial` objects and class-based callables.
+    This is the Python 3 version where getargspec is depreciated and replaced by
```

Comment:
Typo here: `depreciated` -> `deprecated`. FWIW, I don't think the docstring needs to talk about `getargspec` vs. `inspect.signature` because that is getting too far into implementation details.

Perhaps we should only document `get_func_args`, and leave the version-specific functions "private". Something like:

```python
def _get_func_args2(func):
     # ... no docstring ...

def _get_func_args3(func):
    # ... no docstring ...

#: Given a callable, return a list of argument names.
#: Handles `functools.partial` objects and callable objects.
get_func_args = _get_func_args2 if PY2 else _get_func_args3